client/index.html

```diff
@@ -222,7 +222,7 @@
               <div class="sidenav md-toolbar-tools">
                 <h2 class="menu-heading menu-subhead">
                 <span>
-                  <md-icon class=".md-default-theme md-warn">{{vm.menu.locationStateIcon}}</md-icon>
+                  <md-icon ng-hide="vm.menu.locationStateIcon == undefined" class=".md-default-theme md-warn">{{vm.menu.locationStateIcon}}</md-icon>
```

Comment:
Change this to ng-if vm.menu !== undefined.

This way, the icon won't show initially and will only appear if the statement is true